### PR TITLE
Fixing kerberos package dependency issue

### PR DESCRIPTION
**Pull request by kenshaw**
A number of fixes, as itemized:

- Change import paths for github.com/jcmturner/gokrb5/* -> gopkg.in/jcmturner/gokrb5.v1/*
- Update Gopkg.lock accordingly
- Adding line end to files missing them
- Reorder import list on class_mappings.go
- Fix gen-protobuf.sh script and permissions
- Generate latest message/*.pb.go files

### CODE REVIEW

**Comment by kenshaw** [comment]
I'm humbling requesting this PR be merged in order to re-add this SQL driver to `usql` (https://github.com/xo/usql). I was forced to remove this driver completely as the package path for the kerberos package changed (now using gopkg.in), and caused this dependency to no longer be `go get`-able.

Additionally, I did some other minor cleanup and regenerated the protobufs based on the latest stable protoc/protoc-gen-go binaries.

File: .gitignore
Comment (F21) [inline .gitignore]: I think we can remove the `/calcite-tmp/` directory from the .gitignore because we now use the `avatica-tmp` directory.

**Review by F21 — CHANGES_REQUESTED** [review]
Just a minor comment!

**Comment by F21** [comment]
@kenshaw Thank you so much for this PR! Just a small comment in the review and it's good to be merged! 👍

File: gen-protobuf.sh
Comment (F21) [inline gen-protobuf.sh]: Also, another question, why do we need to do this here?

File: gen-protobuf.sh
Comment (F21) [inline gen-protobuf.sh]: Also, another question, why do we need to do this here?

Comment (kenshaw) [reply gen-protobuf.sh]: I'm not sure what version of `protoc` that was in use previously, but without specifying the `go_package` option, you get an error like this:

```
2017/10/16 13:12:23 protoc-gen-go: error:inconsistent package names: requests common
```

There might be a better/easier way to fix this via a passed command line option, but the most expedient/easiest thing I found to do was to just manually set the `go_package` option in the protobuf files.

**Comment by kenshaw** [comment]
I changed the `.gitignore` as requested, and rebased/pushed the changes to the PR.

File: gen-protobuf.sh
Comment (F21) [reply gen-protobuf.sh]: hey @kenshaw,

Thanks for investigating. I did some testing and the following works with protoc 3.4.0:

```
protoc --proto_path=avatica-tmp/core/src/main/protobuf --go_out=import_path=message:message avatica-tmp/core/src/main/protobuf/*.proto
```
Can you update the PR to use this? We should be good to merge after this change 👍

Comment (kenshaw) [reply gen-protobuf.sh]: Done.

**Review by F21 — APPROVED** [review]

**Comment by F21** [comment]
@kenshaw Thanks! Just waiting for the tests to turn green before merging.

**Comment by F21** [comment]
Released as `v2.2.0`.

**Pull request merged**
